DotNetCasClient/Validation/CasSaml11Response.cs

```diff
@@ -98,7 +98,7 @@ private void ProcessValidAssertion()
         {
             protoLogger.Debug("Unmarshalling SAML response");
             XmlDocument document = new XmlDocument();
-            document.Load(new StringReader(_CasResponse));
+            document.LoadXml(_CasResponse);

```

Comment:
I was just looking through sources, and things like `new StreamReader` without `using` immediately raise red flag in my mind; to replace `Load(Stream)` with `LoadXml(string)` is somehow obvious step to me.